packages/react-avatar/src/components/AvatarGroup/AvatarGroup.types.ts

```diff
@@ -1,19 +1,72 @@
 import type { ComponentProps, ComponentState, Slot } from '@fluentui/react-utilities';
+import { PopoverSurface } from '@fluentui/react-popover';
+import { AvatarSizes } from '../Avatar/Avatar.types';
+import { Button } from '@fluentui/react-button';
+import { TooltipProps } from '@fluentui/react-tooltip';
 
 export type AvatarGroupSlots = {
   root: Slot<'div'>;
+
+  /**
+   * Popover trigger slot that can be used to change the overflow indicator.
+   */
+  popoverTrigger?: Slot<typeof Button>;
+
+  /**
+   * Popover surface that will be displayed when the popover is triggered.
+   */
+  popoverSurface?: Slot<typeof PopoverSurface>;
 };
 
 type AvatarGroupCommons = {
-  // TODO Add things shared between props and state here
+  /**
+   * Layout the Avatars should be displayed as.
+   * @default grid
+   */
+  layout: 'grid' | 'stacked' | 'pie';
```

Comment:
What do you think about `stack` instead of `stacked`?

Reply:
My thinking is that "grid" and "pie" are nouns while "stacked" is an adjective and these should all be the same part of speech. I know it's a bit nitpicky but it brings a nice consistency to naming.

Reply:
We will go with the following naming: `spread`, `stack`, and `pie`